nexus/db-queries/src/db/datastore/inventory.rs

```diff
@@ -1266,9 +1453,14 @@ impl DataStore {
             ndatasets,
             nphysical_disks,
             nnvme_disk_disk_firmware,
-            nsled_agent_zones,
-            nzones,
-            nnics,
+            nlast_reconciliation_disk_results,
+            nlast_reconciliation_dataset_results,
+            nlast_reconciliation_zone_results,
+            nomicron_sled_configs,
+            nomicron_sled_config_disks,
+            nomicron_sled_config_datasets,
+            nomicron_sled_config_zones,
+            nomicron_sled_config_zone_nics,
             nzpools,
             nerrors,
             nclickhouse_keeper_membership,
```

Comment:
would strongly prefer this to be a typed structure rather than a ~20 tuple, to ensure that names don't get swapped around by accident.

Reply:
Great call. Didn't find any out-of-order fields, but did find a few typos: 577483b82